test/e2e/apps/statefulset.go

```diff
@@ -901,21 +913,29 @@ var _ = SIGDescribe("StatefulSet", func() {
 			framework.DeleteAllStatefulSets(c, ns)
 		})
 
+		// This can't be Conformance yet because it depends on a default
```

Comment:
This should never be conformance. We should consider removing it.

Reply:
Is it because they are application-specific, or because they rely on dynamic volume provisioning, which is an optional capability?

Reply:
Ken's comment was originally on the ZooKeeper example, and he was referring only to the app-specific tests. I rewrote the comments on those to explain that they are excluded for being app-specific.

After the squash and force push, GitHub has mistakenly placed his comment on a totally different line (that happens to have the same contents as the line he previously commented on).

The "should provide basic identity" test was never in question for being app-specific. It is only excluded because it relies on dynamic provisioning and a default storage class, as indicated in the comment.